newsroom/utils.py

```diff
@@ -519,6 +519,13 @@ def format_date(date, time, offset):
         return _today - timedelta(days=7)
     if date == "now-30d/d":
         return _today - timedelta(days=30)
+    if date == "now30d/d":
```

Comment:
I wonder where those values are coming from, seems like they are wrong filters

Reply:
we defined here check this PR:- https://github.com/superdesk/newsroom-app-cp/pull/186

Reply:
seems like a wrong PR link

Reply:
https://github.com/superdesk/newsroom-app-cp/pull/186/files#diff-d8def684c9a77cfaaa19b4c1762a220d5978a030144169de2a502eedd110651bR419

Reply:
I see, might be better to define some descriptive `filter` param then and resolve the query only on server, we don't need it in url